front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,347 @@
+import React, { useMemo } from 'react';
+
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { Page, Text, Image, Document, View } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import iconAlert from 'assets/simulationSheet/icon_alert_fill.png';
+import logoSNCF from 'assets/simulationSheet/logo_sncf_reseau.png';
+import type { PostStdcmApiResponse, RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+import { formatDay } from 'utils/date';
+import { getStopDurationTime, getStopTime } from 'utils/timeManipulation';
+
+import { fakeInformation, styles } from './SimulationReportStyleSheet';
+import { formatCreationDate } from '../utils';
+
+type SimulationReportSheetProps = {
+  stdcmData: PostStdcmApiResponse;
+  rollingStockData: RollingStockWithLiveries;
+  number: string;
+  mapCanvas?: string;
+};
+
+const SimulationReportSheet = ({
+  stdcmData,
+  rollingStockData,
+  number,
+  mapCanvas,
+}: SimulationReportSheetProps) => {
+  const { t } = useTranslation('stdcm-simulation-sheet');
+  const formattedParts = formatCreationDate(stdcmData.path.created);
+  const formattedDate = t('formattedDate', {
+    day: formattedParts.formattedDay,
+    month: formattedParts.formattedMonth,
+    year: formattedParts.year,
+    hours: formattedParts.formattedHours,
+    minutes: formattedParts.formattedMinutes,
+  });
```

Comment:
I think it's more concise :)
```suggestion
  const creationDate = t('formattedDate', formatCreationDate(stdcmData.path.created));
```

To do so, you need to modify what formatCreationDate returns:
```
  return {
    day: formattedDay,
    month: formattedMonth,
    year,
    hours: formattedHours,
    minutes: formattedMinutes,
  };
```

Reply:
nice ! fixed